packages/core/useGeolocation/index.ts

```diff
@@ -7,7 +7,9 @@ import type { ConfigurableNavigator } from '../_configurable'
 import { defaultNavigator } from '../_configurable'
 import { useSupported } from '../useSupported'
 
-export interface UseGeolocationOptions extends Partial<PositionOptions>, ConfigurableNavigator {}
+export interface UseGeolocationOptions extends Partial<PositionOptions>, ConfigurableNavigator {
+  immediate?: boolean
```

Comment:
Is this doing anything?  I don't see it being used

Reply:
From the code in #2372 

We are missing:

```
if (immediate)
    resume()
```
so resume() is not always directly called. I can make a PR as I need that feature.